.changeset/pink-gifts-retire.md

```diff
@@ -0,0 +1,7 @@
+---
+"@clerk/clerk-js": patch
+"@clerk/clerk-react": patch
+"@clerk/shared": patch
```

Comment:
🙃 i think we should make it a `minor` since we have introduced a new export `import { isDeeplyEqual } from '@clerk/shared/react';`

Reply:
done